### PR TITLE
Add chartjs support

### PR DESCRIPTION
**Pull request by dan-nectry**
Lots of chart types should be possible, but `SimpleGraph` still only supports bar charts.

### CODE REVIEW

File: lib.urp [hunk 1]
```diff
@@ -3,6 +3,7 @@ library $/bootstrap/bootstrap5
 library $/ajaxUpload
 library ckeditor
 library chartist
```
Comment (achlipala-biz) [inline lib.urp]: It probably isn't worth leaving multiple chart libraries in this library.  How would you feel about removing Chartist?

**Review by achlipala-biz — COMMENTED** [review]
I just noticed one small but potentially worthwhile opportunity for cleanup.

File: lib.urp (hunk 1 above)
Comment (dan-nectry) [reply lib.urp]: Happy to do it.

**Comment by dan-nectry** [comment]
Actually, please wait on merging this.  I have a slight tweak I'd like to make.

**Comment by dan-nectry** [comment]
Okay, now it's ready.  `SimpleGraph` now takes an argument to specify which simple graph type you want to use (e.g., Bar, Pie, etc.)

**Pull request merged**